assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -384,8 +384,27 @@ che.infra.kubernetes.ingress.domain=
 # to create new namespace must be used.
 #
 # Ignored for OpenShift infra. Use `che.infra.openshift.project` instead
+#
 che.infra.kubernetes.namespace=
 
+# Defines Kubernetes default namespace in which user's workspaces are created
+# if user does not override it.
+# It's possible to use <username> and <userid> placeholders (e.g.: che-workspace-<username>).
```

Comment:
It's always Che user but depending on configuration it may be the same as 
Keycloak, OpenShift, Github, Gitlab username...
Will think more and maybe add some info but I would like to keep this description simple, without authentication-related stuff

Reply:
Sounds good, I'm mostly concerned from an admin point of view, would be nice to have an idea of what `<user>-che` would look like.

As an aside, can this cause issues if user creates an account that conflicts with a k8s-reserved namespace? E.g. with
- `che.infra.kubernetes.namespace.default=<username>` and username `default`
- `che.infra.kubernetes.namespace.default=<username>-che` and username `kubernetes-anything`

I don't know if this is worth documenting or if it would be considered common sense.

Reply:
`che.properties` is not a place for user-oriented documentation.
I think it's OK to miss such info here and consider adding extra info about authentication in Che 7 Doc.